From d99c48d276c383ba688adced498c4eb5d9797afa Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Fri, 26 Feb 2021 07:15:53 +0100
Subject: [PATCH] treewide: use new extra_command function

In OpenWrt 21.xx there is a used new extra_command function which
replaced the old one. Even though, the old one is still compatible,
let's use a new one to be in sync with upstream
---
 net/atlas-sw-probe/Makefile            |  2 +-
 net/atlas-sw-probe/files/atlas.init    | 13 ++++++-------
 net/knot-resolver/Makefile             |  2 +-
 net/knot-resolver/files/kresd.init     |  2 +-
 net/suricata-pakon/Makefile            |  4 ++--
 net/suricata-pakon/files/suricata.init |  3 +--
 6 files changed, 12 insertions(+), 14 deletions(-)

diff --git a/net/atlas-sw-probe/Makefile b/net/atlas-sw-probe/Makefile
index f23dad786..c8e50106b 100644
--- a/net/atlas-sw-probe/Makefile
+++ b/net/atlas-sw-probe/Makefile
@@ -8,7 +8,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=atlas-sw-probe
 PKG_VERSION:=5020
-PKG_RELEASE:=3
+PKG_RELEASE:=4
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/ripe-atlas-software-probe
diff --git a/net/atlas-sw-probe/files/atlas.init b/net/atlas-sw-probe/files/atlas.init
index c5495cd9a..53105ff84 100644
--- a/net/atlas-sw-probe/files/atlas.init
+++ b/net/atlas-sw-probe/files/atlas.init
@@ -2,13 +2,12 @@
 
 USE_PROCD=1
 START=30
-EXTRA_COMMANDS="get_key probeid log create_backup load_backup"
-EXTRA_HELP="	get_key	print probe public key (used for probe registration)
-	probeid	print probe id
-	log	print probe status log
-	create_backup 	backup ssh key to tar.gz
-	load_backup 'backup.tar.gz' 	load backup ssh key from tar.gz
-"
+
+extra_command "get_key" "Print probe public key (used for probe registration)"
+extra_command "probeid" "Show probe ID"
+extra_command "log" "Show status log"
+extra_command "create_backup" "Backup SSH key to .tar.gz"
+extra_command "load_backup" "Load backup SSH key from .tar.gz"
 
 SCRIPTS_DIR="/usr/libexec/atlas-probe-scripts"
 TMP_BASE_DIR="/tmp/ripe_atlas_probe"
diff --git a/net/knot-resolver/Makefile b/net/knot-resolver/Makefile
index 3a93ce3de..f78de8e7f 100644
--- a/net/knot-resolver/Makefile
+++ b/net/knot-resolver/Makefile
@@ -11,7 +11,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=knot-resolver
 PKG_VERSION:=5.2.1
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://secure.nic.cz/files/knot-resolver
diff --git a/net/knot-resolver/files/kresd.init b/net/knot-resolver/files/kresd.init
index 52d99c76d..d55031c64 100755
--- a/net/knot-resolver/files/kresd.init
+++ b/net/knot-resolver/files/kresd.init
@@ -6,7 +6,7 @@ START=61
 STOP=00
 
 USE_PROCD=1
-EXTRA_COMMANDS="test_ipv6"
+extra_command "test_ipv6" "Test IPv6 connection"
 PROG=/usr/sbin/kresd
 CONFIGFILE=/tmp/kresd.config
 ROOTKEYFILE=/etc/root.keys
diff --git a/net/suricata-pakon/Makefile b/net/suricata-pakon/Makefile
index 135faa35b..811be4766 100644
--- a/net/suricata-pakon/Makefile
+++ b/net/suricata-pakon/Makefile
@@ -8,8 +8,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=suricata-pakon
-PKG_VERSION:=1
-PKG_RELEASE:=8
+PKG_VERSION:=1.0
+PKG_RELEASE:=9
 
 PKG_LICENSE:=GPL-2.0
 PKG_INSTALL:=1
diff --git a/net/suricata-pakon/files/suricata.init b/net/suricata-pakon/files/suricata.init
index 6c3962bb4..c4ae7fad4 100644
--- a/net/suricata-pakon/files/suricata.init
+++ b/net/suricata-pakon/files/suricata.init
@@ -3,8 +3,7 @@
 USE_PROCD=1
 START=95
 STOP=05
-EXTRA_COMMANDS="status"
-EXTRA_HELP="        status Get suricata status (as exit code)"
+extra_command "status" "Display the service status (as exit code)"
 IPTABLES_RULES="/tmp/suricata/suricata.fw"
 IP6TABLES_RULES="/tmp/suricata/suricata.fw6"
 PID_FILE="/var/run/suricata/suricata.pid"
-- 
2.25.1

